amenity-points.mss

```diff
@@ -1906,7 +1913,8 @@
     }
   }
 
-  [feature = 'amenity_atm'][zoom >= 17] {
+  [feature = 'amenity_atm'][zoom >= 17],
+  [feature = 'amenity_bbq'][zoom >= 17] {
     text-name: "[operator]";
```

Comment:
Name tag is used here, operator is not even visible in stats:

https://taginfo.openstreetmap.org/tags/amenity=bbq#combinations

Reply:
Something went wrong and - just by looking at code - you render operator instead of name again.